src/libs/ajax.js

```diff
@@ -64,7 +64,7 @@ const Sam = {
       _.merge(authOpts(), jsonBody(scopes), { method: 'POST' })
     )
       .then(parseJson)
-  }, 1000 * 60 * 30)
+  }, namespace => namespace, 1000 * 60 * 30)
```

Comment:
`_.identity`?

Reply:
Technically yes, but part of the concern was that key generation wasn't explicit, so I was trying to be extra explicit.

Reply:
Agreed. I might also suggest a comment saying exactly that to prevent removal of this clarity by some future refactoring.

Reply:
Point taken, will do next time.